tests/arbitrary-variants.test.js

```diff
@@ -643,18 +643,18 @@ crosscheck(({ stable, oxide }) => {
           raw: html`
             <div>
               <div class="aria-checked:underline"></div>
-              <div class="aria-[sort=ascending]:underline"></div>
-              <div class="aria-[labelledby=a_b]:underline"></div>
+              <div class="aria-[sort="ascending"]:underline"></div>
+              <div class="aria-[labelledby="a_b"]:underline"></div>
               <div class="group-aria-checked:underline"></div>
               <div class="peer-aria-checked:underline"></div>
               <div class="group-aria-checked/foo:underline"></div>
               <div class="peer-aria-checked/foo:underline"></div>
-              <div class="group-aria-[sort=ascending]:underline"></div>
-              <div class="peer-aria-[sort=ascending]:underline"></div>
-              <div class="group-aria-[labelledby=a_b]:underline"></div>
-              <div class="peer-aria-[labelledby=a_b]:underline"></div>
-              <div class="group-aria-[sort=ascending]/foo:underline"></div>
-              <div class="peer-aria-[sort=ascending]/foo:underline"></div>
+              <div class="group-aria-[sort="ascending"]:underline"></div>
+              <div class="peer-aria-[sort="ascending"]:underline"></div>
+              <div class="group-aria-[labelledby="a_b"]:underline"></div>
+              <div class="peer-aria-[labelledby="a_b"]:underline"></div>
+              <div class="group-aria-[sort=:"ascending"]/foo:underline"></div>
+              <div class="peer-aria-[sort="ascending"]/foo:underline"></div>
```

Comment:
While this technically works in the test it would be invalid HTML. Should you change to single quotes?

Reply:
Yep thanks! I removed the quotes again in most cases, but added them for the once where it was required like `foo_bar` cases.